### DISCUSSION TITLE
Feature Request: `.to_polars` or `.to_df(library='polars')`?

### DISCUSSION THREAD

**Discussion by MarcoGorelli** [duckdb/duckdb#14146] [Ideas]
I'd like to be able to operate with DuckDB on a Polars dataframe and then convert straight back to Polars, without having to explicitly go via pandas or PyArrow (no objections is that happens under the hood)

Would you consider adding either `.to_polars`, or adding a `library` argument to `to_df`? OK if it defaults to `library='pandas'` for backwards compatibility, but being able to go straight to Polars would be really nice

**Comment by Maxxen** [comment]
I think you can already do this with the .pl() method https://duckdb.org/docs/guides/python/polars.html#duckdb-to-polars

**Reply by MarcoGorelli** [reply]
Thanks, sorry for missing that
